portfolio/src/main/webapp/script.js

```diff
@@ -40,36 +40,36 @@ async function getHello() {
  *Adds json messages to the message division
  */
 async function getMessage() {
-    const response = await fetch('/data');
-    const json = await response.json();
-    const message = "" + json.msg1 + " " + json.msg2 + " " + json.msg3;
-    document.getElementById('message-container').innerText = message;
+  const response = await fetch('/data');
+  const json = await response.json();
+  const message = "" + json.msg1 + " " + json.msg2 + " " + json.msg3;
+  document.getElementById('message-container').innerText = message;
 }
 
 /**
  *Gets the comments posted and prints them back
  */
 async function getComments() {
   var max_comments = document.getElementById('max-comment').value;
-  const response = await fetch('/data?max-comment='+max_comments);
+  const response = await fetch('/data?max-comment=' + max_comments);
   const json = await response.json();
   const posted_comments = document.getElementById('posted_comments');
```

Comment:
Nit: JS style guide also suggests using lowerCamelCase for local variables (https://google.github.io/styleguide/jsguide.html#naming-local-variable-names)

Reply:
Changed the variable name to maxComments

Reply:
Please also rename `posted_comments`